VotingApplication/VotingApplication.Web/Views/Create/ConfirmStep.cshtml

```diff
@@ -118,10 +121,10 @@
             <div layout="row">
                 <md-button ng-hide="loading"
                            ng-disabled="pollHasErrors()"
-                           ng-click="loading=!loading"
-                           ng-class="{'md-warn' : pollHasWarnings()}"
+                           ng-click="loading=!loading;createPoll()"
```

Comment:
This should be done in a single method in the controller, not as consecutive statements in an `ng-click`.